site/content/en/docs/Installation/helm.md

```diff
@@ -129,14 +129,17 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `agones.controller.healthCheck.timeoutSeconds`      | Number of seconds after which the probe times out (in seconds)                                  | `1`                    |
 | `agones.controller.resources`                       | Controller resource requests/limit                                                              | `{}`                   |
 | `agones.controller.generateTLS`                     | Set to true to generate TLS certificates or false to provide your own certificates in `certs/*` | `true`                 |
+| `agones.controller.numWorkers`                      | Number of workers to spin per resource type                                                     | `64`                   |
```

Comment:
This section should be below in the "new configuration features" section. (best way I could handle it in the docs)

Reply:
done